woocommerce/woocommerce
# Hide In-app marketplace feature flag UI

### Changes proposed in this Pull Request:

The new React in-app marketplace UI has been in place since WooCommerce 8.3 and is now ready to be active on all sites. This PR removed the UI to disable the feature flag. For now, the feature flag still exists, but you will no longer be able to toggle it.

Closes https://github.com/Automattic/woocommerce.com/issues/19087



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch
2. Visit **WooCommerce > Settings > Advanced > Features** and see that the option to disable the marketplace is no longer visible
3. Test around this issue



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Removed the UI to disable React marketplace interface.

</details>
